File: src/main/java-graphql/io/github/graphql/java/resolver/ProjectOwnerResolver.java
package io.github.graphql.java.resolver;

import io.github.graphql.java.model.*;

/**
 * Resolver for ProjectOwner
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public interface ProjectOwnerResolver {

    ProjectTO project(ProjectOwnerTO projectOwnerTO, int number) throws Exception;

    @javax.validation.constraints.NotNull
    ProjectConnectionTO projects(ProjectOwnerTO projectOwnerTO, String after, String before, Integer first, Integer last, ProjectOrderTO orderBy, String search, java.util.List<ProjectStateTO> states) throws Exception;

}

File: src/main/graphql/io/github/graphql/resolver/ProjectOwnerResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for ProjectOwner
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface ProjectOwnerResolver {

    @Throws(Exception::class)
    fun project(projectOwnerTO: ProjectOwnerTO, number: Int): ProjectTO?

    @Throws(Exception::class)
    fun projects(projectOwnerTO: ProjectOwnerTO, after: String?, before: String?, first: Int?, last: Int?, orderBy: ProjectOrderTO?, search: String?, states: List<ProjectStateTO>?): ProjectConnectionTO

}
